sys-apps/util-linux/patches/util_linux-2.31-x86.patchset

```diff
@@ -0,0 +1,54 @@
+From e6d29881ba8f65f0362d3b2f85fc3217b0143355 Mon Sep 17 00:00:00 2001
+From: fbrosson <fbrosson@localhost>
+Date: Sat, 13 Jan 2018 13:33:12 +0000
+Subject: skip the "scanf_cv_alloc_modifier" configure test on x86.
+
+It's not required and it crashes conftest.
+
+diff --git a/configure.ac b/configure.ac
+index 1899ec3..d03f518 100644
+--- a/configure.ac
++++ b/configure.ac
+@@ -601,39 +601,6 @@ int main()
+ 	return rc;
+ }])
+ 
+-AC_CACHE_VAL([scanf_cv_alloc_modifier],
```

Comment:
Since it's cached, maybe all you need is to add `scanf_cv_alloc_modifier=no` to `./configure` and it will happily skip the test.